src/averageAges.js

```diff
@@ -14,12 +14,34 @@
  *
  * @return {number}
  */
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+const CENTURY_VALUE = 100;
+
+const getAverageAge = (peopleArray) =>
+  peopleArray.reduce((acc, person, personIndex) => {
+    const { born, died } = person;
+    const age = died - born;
+
+    if (personIndex === peopleArray.length - 1) {
+      return (acc + age) / peopleArray.length;
+    }
+
+    return acc + age;
+  }, 0);
```

Comment:
The 'getAverageAge' function is well-implemented, but consider using it with a condition instead of checking for the last personIndex in the main functions. It could make the code more concise.